assets/stories/skate-components/pages/app.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import React from "react"
+import { MemoryRouter } from "react-router-dom"
+
+import { AppRoutes } from "../../../src/components/app"
+import { AppStateProvider } from "../../../src/components/appStateWrapper"
+
+const meta = {
+  component: AppRoutes,
+  parameters: {
+    layout: "fullscreen",
+    stretch: true,
+  },
+  decorators: [
+    (StoryFn) => (
+      <MemoryRouter initialEntries={["/"]}>
+        <StoryFn />
+      </MemoryRouter>
+    ),
+    (StoryFn) => (
+      <AppStateProvider>
+        <StoryFn />
+      </AppStateProvider>
+    ),
```

Comment:
**question (non-blocking, just curious):** do you see a specific advantage to chaining decorators rather than having a single decorator that wraps the story in both the memory router and app state providers?

Reply:
I was having trouble deciding on this.
On one hand, I think doing them "chained" better fits the "single responsibility principle", but that's more a coincidence than a intention. I did it like this to make diff's nicer, but couldn't decide if I wanted them merged or not.

Reply:
I'm leaning toward leaving it, just because it keeps indentation lower, do you have any preferences on this before I merge?
cc: @joshlarson

Reply:
I'm fine with how it is! Was mainly just wondering what your thoughts were.